prow/tide/codereview.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Are we planning on only using this for Tide? If not is this still the place we want to keep this?

Reply:
At least for this work(gerrit tide) this is only used for Tide. I think it makes more sense to keep it here in this case, to avoid confusion